pokemonEntryTemplate.phtml

```diff
@@ -0,0 +1,17 @@
+<li class="poke-container">
```

Comment:
I recall that the styling for seen/caught pokemon is done on this element. Have you accounted for this or is it part of another task?